test/protocol-tests/src/test/java/software/amazon/awssdk/protocol/tests/timeout/BaseApiCallAttemptTimeoutTest.java

```diff
@@ -15,126 +15,75 @@
 
 package software.amazon.awssdk.protocol.tests.timeout;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
-import static com.github.tomakehurst.wiremock.client.WireMock.anyUrl;
-import static com.github.tomakehurst.wiremock.client.WireMock.post;
-import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
-import static software.amazon.awssdk.protocol.wiremock.WireMockUtils.verifyRequestCount;
 
-import com.github.tomakehurst.wiremock.stubbing.Scenario;
+import java.time.Duration;
 import org.junit.Test;
+import software.amazon.awssdk.testutils.service.http.MockHttpClient;
+import software.amazon.awssdk.utils.Pair;
 
 public abstract class BaseApiCallAttemptTimeoutTest extends BaseTimeoutTest {
 
-    protected static final int API_CALL_ATTEMPT_TIMEOUT = 800;
-    protected static final int DELAY_BEFORE_API_CALL_ATTEMPT_TIMEOUT = 100;
-    protected static final int DELAY_AFTER_API_CALL_ATTEMPT_TIMEOUT = 1000;
+    protected static final Duration API_CALL_ATTEMPT_TIMEOUT = Duration.ofMillis(100);
```

Comment:
Are you lowering these timeout values because the tests are more reliable now, and this improves the time the tests take?

Reply:
Yup, lowering them because they are more reliable now.